src/Features/PaymentGatewaySuggestions/Init.php

```diff
@@ -105,27 +105,6 @@ public static function localize( $specs ) {
 
 			$data = (object) array_merge( (array) $locale, (array) $spec );
 			unset( $data->locales );
-
-			$data->fields = array();
-
-			// Loop over and localize fields.
-			foreach ( $spec->fields as $field ) {
-				if ( ! isset( $field->locales ) ) {
-					continue;
-				}
-
-				$locale = SpecRunner::get_locale( $field->locales );
-
-				if ( ! $locale ) {
-					continue;
-				}
-
-				$field_data = (object) array_merge( (array) $field, (array) $locale );
-				unset( $field_data->locale );
-				unset( $field_data->locales );
-				$data->fields[] = $field_data;
-			}
```

Comment:
I assume this isn't relevant anymore?

Reply:
Correct, fields are now handled directly from the payment gateway and passed via the REST API.